lib/features/navigator/utils/extract_tabbar_arg.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/widgets.dart';
+
+import '../../bottom_nav_bar/nav_bar_config.dart';
+
+extension ExtractTabbarArg on RouteSettings? {
+  NavBarEnum? get selectedTab {
+    final args = this?.arguments;
+    if (args is NavBarEnum) {
+      return args;
+    }
+    return null;
```

Comment:
I think it's good to change it to sth like this (we stick to that convention in other files and in my opinion it's more readable)
```dart
NavBarEnum? get selectedTab {
    final args = this?.arguments;
    return args is NavBarEnum ? args : null;
  }
```

Reply:
matter of preferences I guess, but suggestion applied :)